packages/frontend/src/components/SavedDashboards/AddTilesToDashboardModal.tsx

```diff
@@ -247,10 +248,13 @@ const AddTilesToDashboardModal: FC<AddTilesToDashboardModalProps> = ({
                                 required
                                 {...form.getInputProps('dashboardName')}
                             />
-                            <TextInput
+                            <Textarea
                                 id="dashboard-description"
                                 label="Dashboard description"
                                 placeholder="A few words to give your team some context"
+                                autosize
+                                maxRows={3}
+                                style={{ overflowY: 'auto' }}
```

Comment:
do we need this @jogeshwar01 ?

Reply:
@joaoviana Its more of a UI/UX thing.
Basically If we skip these, the only thing that's an issue is a default 2 liner textarea even if there is single line of description. Like this :
![image](https://github.com/lightdash/lightdash/assets/85165953/7ba88b9d-57da-4916-8a72-750cef5d0320)

The `autosize` and `maxRows` helps us counter this. We can skip the overflowY:'auto' here.

I would love your opinion on which option we should choose here. Will update the PR accordingly.

Reply:
Can you remove the `style={{ overflowY: 'auto' }}` everywhere then @jogeshwar01 ?